src/Core/Utilities/JsonHelpers.cs

```diff
@@ -129,4 +130,63 @@ public override void Write(Utf8JsonWriter writer, DateTime? value, JsonSerialize
             writer.WriteStringValue(CoreHelpers.ToEpocMilliseconds(value.Value).ToString());
         }
     }
+
+    /// <summary>
+    /// Allows reading a string from a number or string, should only be used on string properties
+    /// </summary>
+    public class PermissiveStringConverter : JsonConverter<string>
+    {
+        internal static PermissiveStringConverter Instance = new();
+
+        public override string Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            return reader.TokenType switch
+            {
+                JsonTokenType.String => reader.GetString(),
+                JsonTokenType.Number => reader.GetInt64().ToString(),
```

Comment:
Very good idea, should I also check for `UInt64` in between `Int64` and `Decimal`?

Reply:
Decimal has a larger MaxValue than what UInt64 so if UInt64 is last I don't think it would ever run. 

After looking at how the .ToString() on decimal prints values depending on the input I think we actually could just use `reader.GetDecimal().ToString()` and get everything we want. https://dotnetfiddle.net/0pS02M

Reply:
Okay, yeah, good point, let's just use `Decimal` then for numeric